test/create-element-extension/create-element-extension.html

```diff
@@ -0,0 +1,10 @@
+<script>
+  class Foo {
+    handler(event) {
+      document.createElement('style', 'custom-style');
```

Comment:
I'm not familiar with this API. Was passing in a second string argument a supported API in v0?

Reply:
I believe Polymer implemented it. https://github.com/w3c/webcomponents/issues/544

Reply:
Arg, I see, gotcha